Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -252,6 +258,58 @@ def ip_to_indicator_type(ip):
             return None
 
 
+def auto_detect_indicator_type(indicator_value):
+    """
+      Infer the type of the indicator.
+
+      :type indicator_value: ``str``
+      :param indicator_value: The indicator whose type we want to check. (required)
+
+      :return: The type of the indicator.
+      :rtype: ``str``
+    """
+    if re.match(ipv4cidrRegex, indicator_value):
```

Comment:
how about moving the import to here and if we get ImportError then return proper error that explains the user that if he wants to use auto_detect_indicator_type function he should use docker image that contains tldextract module and give an example to the dockerimage we have